frontend/scss/components/badges.scss

```diff
@@ -0,0 +1,34 @@
+span {
+    &.badge {
+        font-size: 80%;
+    }
+    &.badge-eboard {
+        color: #fff;
+        background-color: #4CAF50;
+    }
+
+    &.badge-rtp {
+        color: #fff;
+        background-color: #ff9800;
+    }
+
+    &.badge-three_da {
+        color: #fff;
+        background-color: #e83e8c;
+    }
+
+    &.badge-webmaster {
+        color: #fff;
+        background-color: #2196F3;
+    }
+
+    &.badge-cm {
+        color: #fff;
+        background-color: #e51c23;
+    }
+
+    &.badge-drink {
+        color: #fff;
+        background-color: #b0197e;
```

Comment:
I'm assuming these colors match the ones on profiles? (I'm too lazy to check)

Reply:
Yes, they are.